spec/models/course_spec.rb

```diff
@@ -5,9 +5,17 @@
   let!(:instance) { create(:instance) }
   with_tenant(:instance) do
     it { is_expected.to belong_to(:creator) }
-    it { is_expected.to have_many(:course_users).inverse_of(:course).dependent(:destroy) }
-    it { is_expected.to have_many(:users).through(:course_users) }
-    it { is_expected.to have_many(:announcements).inverse_of(:course).dependent(:destroy) }
+    it { is_expected.to have_many(:course_users)
+                        .inverse_of(:course)
+                        .dependent(:destroy) }
+    it { is_expected.to have_many(:users)
+                        .through(:course_users) }
+    it { is_expected.to have_many(:announcements)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.